File: app/src/main/java/com/github/shchurov/gitterclient/data/network/responses/MessageResponse.java
package com.github.shchurov.gitterclient.data.network.responses;

import com.google.gson.annotations.SerializedName;

public class MessageResponse {

    public String id;
    public String text;
    @SerializedName("sent")
    public String timeIso;
    @SerializedName("fromUser")
    public UserResponse user;
    public boolean unread;


}

File: app/src/main/kotlin/com/github/shchurov/gitterclient/data/network/responses/MessageResponse.kt
package com.github.shchurov.gitterclient.data.network.responses

import com.google.gson.annotations.SerializedName

class MessageResponse {

    var id: String? = null
    var text: String? = null
    @SerializedName("sent")
    var timeString: String? = null
    @SerializedName("fromUser")
    var user: UserResponse? = null
    var unread: Boolean = false

}